secret-infrastructure/infra/scripts/clear-secret.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I don't think these scripts need to be inside an `infra/` directory. Instead just put them at `helpers/clear-secret.sh` and put documentation in the main README.

Reply:
Moved.